README.md

```diff
@@ -17,8 +40,17 @@ To deploy, create a playbook which looks like this:
             control: "{{ inventory_hostname in groups['cluster_control'] }}"
             batch: "{{ inventory_hostname in groups['cluster_batch'] }}"
             runtime: true
+          openhpc_slurm_service_enabled: true
+          openhpc_slurm_service: slurmd
+          openhpc_slurm_control_host: "{{ groups['cluster_login'] | first }}"
+          openhpc_slurm_partitions: "{{ slurm_compute }}"
```

Comment:
Since we do not specify `slurm_compute`, a user running this example will probably error.

According to the example config from p3-appliances, `slurm_compute` is defined as:

```

slurm_login:
  name: "login"
  flavor: "{{ slurm_login_flavor }}"
  image: "{{ slurm_login_image}}"
  num_nodes: "{{ slurm_login_num_nodes }}"
  user: "centos"
```

Interpolating the variables, we can set the example slurm partitions to something like:
 
```
openhpc_slurm_partitions: 
- name: "compute"
  flavor: "compute-A"
  image: "CentOS7.5-OpenHPC"
  num_nodes: "8"
  user: "centos"
```